src/main/java/lotto/domain/NumberOfManualLottoToPurchase.java

```diff
@@ -0,0 +1,26 @@
+package lotto.domain;
+
+import lotto.exception.InvalidNumberOfManualLottoToPurchase;
+
+public class NumberOfManualLottoToPurchase {
+    private final int numberOfManualLottoToPurchase;
+
+    private NumberOfManualLottoToPurchase(int numberOfManualLottoToPurchase, int maximumNumberOfManualLottoToPurchase) {
+        validateNumberOfManualLottoToPurchase(numberOfManualLottoToPurchase, maximumNumberOfManualLottoToPurchase);
+        this.numberOfManualLottoToPurchase = numberOfManualLottoToPurchase;
+    }
+
+    private void validateNumberOfManualLottoToPurchase(int numberOfManualLottoToPurchase, int maximumNumberOfManualLottoToPurchase) {
+        if(numberOfManualLottoToPurchase < 0 || numberOfManualLottoToPurchase > maximumNumberOfManualLottoToPurchase) {
```

Comment:
전체적으로 코드 포맷팅 확인해주시는 습관 들여주시면 좋을 것 같습니다.
```suggestion
        if (numberOfManualLottoToPurchase < 0 || numberOfManualLottoToPurchase > maximumNumberOfManualLottoToPurchase) {
```